component/src/main/java/com/celements/structEditor/fields/SelectTagPageType.java

```diff
@@ -43,6 +43,7 @@ public void collectAttributes(AttributeBuilder attrBuilder, DocumentReference ce
       attrBuilder.addNonEmptyAttribute("name", getStructDataEditorService().getAttributeName(
           cellDoc, modelContext.getDoc()).or(""));
       if (modelAccess.getFieldValue(cellDoc, FIELD_IS_MULTISELECT).or(false)) {
+        attrBuilder.addEmptyAttribute("multiple");
```

Comment:
Since we still have many XHTML sites, please change to multiple="multiple"

From https://www.w3schools.com/tags/att_select_multiple.asp
In XHTML, attribute minimization is forbidden, and the multiple attribute must be defined as <select multiple="multiple">.